cpp/autosar/src/rules/A7-1-2/VariableMissingConstexpr.ql

```diff
@@ -42,6 +42,12 @@ where
   not v.isConstexpr() and
   not v instanceof Parameter and
   not v.isAffectedByMacro() and
+  // Don't consider non-static member variables.
```

Comment:
As I suggested on the issue (#789) I think we should report non-static member variables, but only if all the member functions of the class have bodies. That should resolve the issue in the example.

Reply:
Done in [d584e30](https://github.com/github/codeql-coding-standards/pull/794/commits/d584e309b212ad29e05e74a4a0760fb567ed92e7)